docs/notebooks/sdba-advanced.ipynb

```diff
@@ -149,6 +149,100 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import xarray as xr\n",
+    "\n",
+    "# Create toy data for the example, here fake temperature timeseries\n",
+    "t = xr.cftime_range('2000-01-01', '2030-12-31', freq='D', calendar='noleap')\n",
+    "ref = xr.DataArray((-20 * np.cos(2 * np.pi * t.dayofyear / 365) + 2 * np.random.random_sample((t.size,)) + 273.15\n",
+    "                    + 0.1 * (t - t[0]).days / 365),  # \"warming\" of 1K per decade,\n",
+    "                   dims=('time',), coords={'time': t}, attrs={'units': 'K'})\n",
+    "sim = xr.DataArray((-18 * np.cos(2 * np.pi * t.dayofyear / 365) + 2 * np.random.random_sample((t.size,)) + 273.15\n",
+    "                    + 0.11 * (t - t[0]).days / 365),  # \"warming\" of 1.1K per decade\n",
+    "                   dims=('time',), coords={'time': t}, attrs={'units': 'K'})\n",
+    "\n",
+    "ref = ref.sel(time=slice(None, '2015-01-01'))\n",
+    "hist = sim.sel(time=slice(None, '2015-01-01'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from xclim.sdba.adjustment import QuantileDeltaMapping\n",
+    "\n",
+    "QDM = QuantileDeltaMapping(nquantiles=15, kind='+', group='time.dayofyear')\n",
+    "QDM.train(ref, hist)\n",
+    "QDM"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The trained `QDM` exposes the training data in the `ds` attribute, we will write to disk, read it back and intialize an new object from it. Notice the `adj_params` in the dataset, that has the same value as the repr string printed just above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "QDM.ds"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "QDM.ds.to_netcdf('QDM_training.nc')\n",
+    "ds = xr.open_dataset('QDM_training.nc')\n",
+    "QDM2 = QuantileDeltaMapping.from_dataset(ds)\n",
+    "QDM2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the case above, creating a full object from the dataset doesn't make the most sense since we are in the same python session, with the \"old\" object still available. This method effective when we reload the training data in a different python session, say on another computer. **However, take note that there is retrocompatiblity insurance.** If the QuantileDeltaMapping class was to change in a new xclim version, one would not be able to create the new object from a dataset saved with the old one.\n",
```

Comment:
```suggestion
    "In the case above, creating a full object from the dataset doesn't make the most sense since we are in the same python session, with the \"old\" object still available. This method effective when we reload the training data in a different python session, say on another computer. **However, take note that there is no retrocompatiblity insurance.** If the QuantileDeltaMapping class was to change in a new xclim version, one would not be able to create the new object from a dataset saved with the old one.\n",
```